domain/domain.go

```diff
@@ -2153,6 +2153,7 @@ func (do *Domain) UpdateTableStatsLoop(ctx, initStatsCtx sessionctx.Context) err
 		do.wg.Run(do.loadStatsWorker, "loadStatsWorker")
 	}
 	owner := do.newOwnerManager(handle.StatsPrompt, handle.StatsOwnerKey)
+	do.wg.Run(func() { quitStatsOwner(do, owner) }, "quitStatsOwner")
```

Comment:
Why do it here?  It seems that we do it in `updateStatsWorkerExitPreprocessing`(in `updateStatsWorker`).

Reply:
If `domain.statsLease` is zero, `updateStatsWorker` is not invoked.

Reply:
Do we need to consider unified handling it, such as the ` updateStatsWorkerExitPreprocessing ` delete inside.